source/community/maintainers.rst

```diff
@@ -108,6 +108,59 @@ Post-Deploy
 * Fast-forward dev branch to the latest master commit
 * Change the wrapper version on the dev branch to ``x.y.zb1``
 
+Images release
+==============
+
+Prepare changes
+---------------
+
+The first step consists in preparing the ``dev`` branch for merge.
+
+1. create a pull request ``dev -> main`` named ``Release X.Y.Z`` (``Release X.Y.ZbI`` is also accepted, X, Y, Z and I being numbers. Creating this pull request will trigger the **pre-release** wrkflows. The PR comment must indicate all major changes.
+
+2. edit the ``dev`` branch until the pull requests checks (pipeline) all pass, effectively publishing all images to the preproduction Dockerhub registry
+
+3. once all checks are good, the PR needs to be approved by a maintainer.
+
+Merge changes
+-------------
+
+Once the PR is approved and ready for merge, it can be merged
+
+1. merge the PR with **Create a merge commit**
+
+2. maintainer then needs to switch to SmartGit/PyCharm/whatever and pull all changes
```

Comment:
Instead of depending on a tools, prefer using git commands to copy and paste (more efficient)
(Cf. Exegol wrapper deployment)